lib/merchant.rb

```diff
@@ -1,9 +1,13 @@
 class Merchant
   attr_reader :id,
-              :name
+              :name,
+              :created_at,
+              :updated_at
 
-  def initialize(hash)
```

Comment:
We should come back and change the name of the argument in initialize from hash to file_path at some point!

Reply:
good reminder @duguy8 !